frontend/src/scenes/sessions/SessionsTable.tsx

```diff
@@ -184,6 +194,20 @@ export function SessionsTable({ personIds, isPersonPage = false }: SessionsTable
             )}
             <PropertyFilters pageKey={'sessions-' + (personIds && JSON.stringify(personIds))} />
 
+            <div className="text-right mb">
+                <Tooltip title={playAllCTA}>
+                    <Button
+                        onClick={() => firstRecordingId && router.actions.push(sessionPlayerUrl(firstRecordingId))}
```

Comment:
Really not a fan of of breaking right-clicking behavior - I'd rather we fixed the tooltip code.

Reply:
Forgot to consider that, let me see what I can do here

Reply:
Done!